### PR TITLE
Cat 1253 security updates

### PR DESCRIPTION
**Pull request by Olusola-Opencast**
Unfortunately the uk.gov.justice.hmpps.gradle-spring-boot package upgrade that deals with the security issue is only beta at the moment.

### CODE REVIEW

File: build.gradle.kts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 plugins {
-  id("uk.gov.justice.hmpps.gradle-spring-boot") version "4.5.7"
-  kotlin("plugin.spring") version "1.7.20"
-  kotlin("plugin.jpa") version "1.7.20"
+  id("uk.gov.justice.hmpps.gradle-spring-boot") version "4.6.0-beta"
```
Comment (kberec) [inline build.gradle.kts:2]: Is it ok to go to a beta version?

**Review by kberec — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (kkonuganti) [reply build.gradle.kts:2]: i would say do not upgrade this yet

Comment (kkonuganti) [reply build.gradle.kts:2]: ok with this as the security update is with in beta as metioned by others.

**Review by kkonuganti — APPROVED** [review]

File: build.gradle.kts (hunk 1 above)
Comment (Olusola-Opencast) [reply build.gradle.kts:2]: Others have done so due to it being the only one that provides a needed security patch

**Pull request merged**